README_v2.md

```diff
@@ -0,0 +1,176 @@
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+
+# Welcome to the Apache Solr project!
+-----------------------------------
+
+Solr is the popular, blazing fast open source enterprise search platform
+written in Java and using [Apache Lucene](https://lucene.apache.org/).
+
+For a complete description of the Solr project, team composition, source
+code repositories, and other details, please see the Solr web site at
+https://solr.apache.org/
+
+## Online Documentation
+
+This README file only contains basic instructions.  For comprehensive documentation,
+visit the [Solr Reference Guide](https://solr.apache.org/guide/).
+
+## Getting Started
+FIXME maybe put the tutorial that Noble or Ishan wrote???   Then at the end put the examples?
+
+### Solr Examples
+
+Solr includes a few examples to help you get started. To run a specific example, enter:
+
+```
+  bin/solr -e <EXAMPLE> where <EXAMPLE> is one of:
+    FIXME the labels in the README do NOT match what is in the bin/solr start -help, and are better!
+    cloud:         SolrCloud example
+    techproducts:  Comprehensive example illustrating many of Solr's core capabilities
+    schemaless:    Schema-less example (schema is inferred from data during indexing)
+    films:         Example of starting with _default configset and adding explicit fields dynamically    
+```
+
+For instance, if you want to run the SolrCloud example, enter:
+
+```
+  bin/solr -e cloud
+```
+
+### Running Solr in Docker
+
+You can run Solr in Docker via the [official image](https://hub.docker.com/_/solr).
+
+To run Solr in a container and expose the Solr port, run:
+
+`docker run -p 8983:8983 solr`
+
+In order to start Solr in cloud mode, run the following.
+
+`docker run -p 8983:8983 solr solr-fg -c`
+
+For documentation on using the official docker builds, please refer to the [DockerHub page](https://hub.docker.com/_/solr).  
+Up to date documentation for running locally built images of this branch can be found in the [local reference guide](solr/solr-ref-guide/src/running-solr-in-docker.adoc).
+
+There is also a gradle task for building custom Solr images from your local checkout.
+These local images are built identically to the official image except for retrieving the Solr artifacts locally instead of from the official release.
+This can be useful for testing out local changes as well as creating custom images for yourself or your organization.
+The task will output the image name to use at the end of the build.
+
+`./gradlew docker`
+
+For more info on building an image, run:
+
+`./gradlew helpDocker`
+
+### Running Solr on Kubernetes
+
+Solr has official support for running on Kubernetes, in the official Docker image.
+Please refer to the [Solr Operator](https://solr.apache.org/operator) home for details, tutorials and instructions.
+
+## Building Solr from Source
+FIXME --> is this where we get JDK from??   What about OpenJDK?
```

Comment:
We are now changing Docker image to use eclipse-temurin, so perhaps we should point people to https://adoptium.net/ here too?

Reply:
Done!